Gemfile

```diff
@@ -9,7 +9,7 @@ gem 'pry'
 # Prevents bundler from taking a long-time to resolve
 group :development, :test do
   gem 'activerecord-import'
-  gem 'mime-types', '~> 1.16'
+  gem 'mime-types', '>= 1.16', '< 4'
```

Comment:
Great! Can you rebase on top of master adding this comment to the commit message? Thanks so much!